xml/snmp_sysdescr.xml

```diff
@@ -3540,6 +3540,7 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="0" name="os.vendor" value="Honeywell"/>
     <param pos="0" name="os.product" value="Thermal Label Printer {hw.model}"/>
     <param pos="0" name="os.device" value="Printer"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:honeywell:intermec_{hw.model}_firmware:{hw.version}"/>
```

Comment:
Unfortunately, modifying the `os.product` value to create alignment with the CPE values won't help us today since `update_cpes.py` does not currently support interpolation in the emitted `*. cpe23` values.